### DISCUSSION TITLE
charts timeout issue.

### DISCUSSION THREAD

**Discussion by allenzhg** [apache/superset#18864] [Q&A / Help] [answered]
When creating a chart, using the data queried by SQL as the data source, an error will be reported if it exceeds 60 seconds. Can this timeout period be modified? What parameters are used specifically?
报错如下：
![image](https://user-images.githubusercontent.com/20509543/155285845-08a2cde8-8914-4f1f-8545-ac8b19438e38.png)

**Comment by villebro** [comment, marked as answer]
Converting to a discsussion under Q&A

**Reply by allenzhg** [reply]
Which god, do you know how to solve it? wait
